magento/magento-cloud-patches
# MAGECLOUD-4606: Review Hotfixes List and Add All Missing for 2.1.4+

### Description
Check all patches from https://magento.com/tech-resources/download for 2.1.4+ versions and bundle missing patches to magento-cloud-patches

### Manual testing scenarios
check edge versions to which patches should be applied
e.g. if patch should be applied for >=2.1.4 <2.2.0, need to check 2.1.4 version and on the latest 2.1 version - 2.1.18.
There is one custom patch - PRODSECBUG-2233  for 2.2.0-2.2.6 versions. It includes fix for PRODSECBUG-2233 and part of fix from https://jira.corp.magento.com/browse/MAGETWO-95236. So, we need to test functionality it fixes on one of 2.2.0-2.2.6 version. Description of it can be taken from https://jira.corp.magento.com/browse/PRODSECBUG-2233

### Contribution checklist
